lib/_tls_wrap.js

```diff
@@ -858,7 +858,8 @@ exports.connect = function(/* [port, host], options, cb */) {
 
   var hostname = options.servername ||
                  options.host ||
-                 options.socket && options.socket._host,
+                 options.socket && options.socket._host ||
```

Comment:
It is unnecessary, as @kkoopa said: `&&` has higher precedence. I just added the `( )` to make the intent clearer to the eye, since the precendence is not obvious.

What's the recommended style for this situation?